bin/pycbc_live

```diff
@@ -378,13 +379,17 @@ with ctx:
                     logging.info("PSD outside acceptable sensitivity %s - %s, have %s",
                                  args.min_psd_abort_distance, args.max_psd_abort_distance, dist)
                     status = False               
-    
+
+            if ifo in followup_ifos:
```

Comment:
Actually thinking about this, we don't even need to use the data readers outside of the master thread. One can do that by checking that evnt.rank == 0 and only advance the followup ones then.